testah-junit/src/test/java/org/testah/MessageDto.java

```diff
@@ -0,0 +1,28 @@
+package org.testah;
+
+public class MessageDto {
+
+    private String type;
+    private String message;
+
+    public MessageDto() {
+
```

Comment:
No need for non-trivial constructor?

Reply:
I usually always include a base constructor, older jackson used to need it.